osu.Game/Skinning/SkinnableSound.cs

```diff
@@ -43,26 +42,19 @@ protected override void SkinChanged(ISkinSource skin, bool allowFallback)
         {
             channels = hitSamples.Select(s =>
             {
-                var ch = loadChannel(s, skin.GetSample);
+                var ch = skin.GetSample(s);
+
                 if (ch == null && allowFallback)
-                    ch = loadChannel(s, audio.Samples.Get);
-                return ch;
-            }).Where(c => c != null).ToArray();
-        }
+                    if (s is HitSampleInfo hsi)
```

Comment:
can you explain why this is used here instead of `lookupnames`?

Reply:
i thought we're not gonna use the lookup names on default skin? are you fine with a duplicate loop from `LegacySkin`?

Reply:
please do.